blog/2023-02-08-preflight-vs-inflight.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "The Great Divide: Preflight vs Inflight Resource Creation ⚔️"
+description: Why does Wing only let you create resources in preflight?
+authors: 
+  - rybickic
+tags: [cloud-oriented programming, winglang, resources, preflight, inflight, iac, security]
+hide_table_of_contents: true
+---
+
```

Comment:
Would it make sense to add a first paragraph that describes what this post is about before diving into the larger context?

Reply:
Added an intro paragraph